### PR TITLE
Make storage POC 2

### PR DESCRIPTION
**Pull request by lykimq**
This is a fork from the PR #38 , this PR add:
- add the `empty_storage`, `make_storage` in `.jsligo` and folder `nft` (`mligo and jsligo`).
- Formatting the code
- Correct spelling `tansfer` to `transfer` in some tests

### CODE REVIEW

File: lib/fa2/asset/extendable_multi_asset.impl.jsligo [hunk 1]
```diff
@@ -1,230 +1,225 @@
 #import "../common/assertions.jsligo" "Assertions"
+
 #import "../common/errors.mligo" "Errors"
+
 #import "../common/tzip12.datatypes.jsligo" "TZIP12"
+
 #import "../common/tzip16.datatypes.jsligo" "TZIP16"
```
Comment (aguillon) [inline lib/fa2/asset/extendable_multi_asset.impl.jsligo]: Is VSCode adding those empty lines?

Comment (lykimq) [reply lib/fa2/asset/extendable_multi_asset.impl.jsligo:8]: Yes.

**Review by aguillon — APPROVED** [review]
Looks good to me, thank you! Also, thank you for fixing the typo in the tests and formatting all the code. I don't use VSCode myself, I hadn't realized that we were so far off.

**Review by Laucans — APPROVED** [review]

**Comment by Laucans** [comment]
LGTM Thanks !

**Pull request merged**